dolphinscheduler-task-plugin/dolphinscheduler-task-http/src/test/java/org/apache/dolphinscheduler/plugin/task/http/HttpParametersTest.java

```diff
@@ -75,4 +81,25 @@ public void testCheckValues() {
         Assert.assertEquals(0,httpParameters.getResourceFilesList().size());
     }
 
+    @Test
+    public void testHttpParams() {
```

Comment:
Could you please use `Assert` in the UT to verified the expected behavior?

Reply:
Have to deal with